src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/terminatedPod.groovy

```diff
@@ -1,10 +1,19 @@
-podTemplate(label: '$NAME', containers: [
-        containerTemplate(name: 'busybox', image: 'busybox', ttyEnabled: true, command: '/bin/cat'),
-    ]) {
-    node ('$NAME') {
+podTemplate(yaml: '''
+spec:
+  containers:
+  - name: busybox
+    image: busybox
+    command:
+    - sleep
+    - 99d
+  terminationGracePeriodSeconds: 3
+''') {
+  retry(count: 2, errorConditions: [kubernetesAgent()]) {
```

Comment:
need to adapt to https://github.com/jenkinsci/workflow-basic-steps-plugin/pull/195/commits/145f716def482946f5f42a1d2f6149fdfb05c6f7